README.md

```diff
@@ -19,13 +19,32 @@ To generate the SDKs supported by this project, you will need:
 
 #### Steps:
 
-From the project root you can run the following commands:
+1. Clone the repository:
+
+```shell
+git clone git@github.com:PhilanthropyDataCommons/sdk.git
+```
+
+2. Change directory:
+
+```shell
+cd sdk
+```
+
+3. Copy your swagger specification
+
+```shell
+cp YOUR_SWAGGER_SPEC_LOCATION ./openapi.json
+```
+
+4. Generate the code bases
 
 ```shell
-> cp YOUR_SWAGGER_SPEC_LOCATION ./openapi.json
 > ./gradlew generateSwaggerCode
 ```
 
 The generated code will now exist in the `build` directory.
-Each language target will have its own directory inside of `build` (e.g. `build/typescript`).
-Some languages might have additional build steps defined in their build's README.md file.
+
+Each language target will have its own directory inside of `build`, and you can follow the readme instructions in that directory with additional steps related to that code base.
```

Comment:
Should add in generic instructions here (at least one sentence / a link to their docs on how to use codegen without using our custom templates)

Reply:
oh this is a great point

(Also whoops -- I had local changes I apparently hadn't pushed before requesting review!  Please check your inbox for an email asking you to re-review).

Reply:
I _only_ make great points.